pandas/io/json/normalize.py

```diff
@@ -25,9 +25,11 @@ def _convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
-def nested_to_record(ds, prefix="", sep=".", level=0):
+def nested_to_record(ds, prefix="", sep=".", level=0,
+                     max_level=None, ignore_keys=[]):
```

Comment:
``ignore_keys=None`` is the idiom. you can assign an empty list if you want *inside* the function